src/ncp/ncp_base.cpp

```diff
@@ -688,7 +758,7 @@ void NcpBase::RegisterPeekPokeDelegates(otNcpDelegateAllowPeekPoke aAllowPeekDel
 
 uint8_t NcpBase::GetWrappedResponseQueueIndex(uint8_t aPosition)
 {
-    while (aPosition >= kResponseQueueSize)
+    while (aPosition >= kResponseQueueSize) // map5: is change needed??
```

Comment:
I see these comments in a few places in this file `// map5: is change needed??`
is this something that kept purposely to address it later? Can we elaborate the comment if that's the case.

Reply:
I forgot to remove those. 
These were places marked for me to look into some more before creating this PR:)